homework1/homework_1_Python.py

```diff
@@ -0,0 +1,51 @@
+def transcribe(sequence):
+    if NA_type == 'RNA':
+        return ('Error, RNA to RNA transcription is not supported, try again')
+    trans_seq = (sequence.replace('T', 'U')).replace('t', 'u')
+    return (trans_seq)
+
+def reverse(sequence):
+    return (sequence[::-1])
+
+def complement(sequence):
+    compl_dic = {'A': 'T', 'T': 'A', 'U': 'A', 'G': 'C', 'C': 'G'}
+    compl_seq = ''
+    if NA_type == 'RNA':
+        compl_dic['A'] = 'U'
+    for nucl in sequence:
+        if nucl.islower():
+            nucl = nucl.upper()
+            compl_seq += compl_dic[nucl].lower()
+        else:
+            compl_seq += compl_dic[nucl]
+    return (compl_seq)
+
+def rev_com(sec):

```

Comment:
```suggestion
def reverse_complement(sec):

```
Не стоит жалеть символов на хорошие названия переменных